app/src/main/java/ru/yandex/practicum/contacts/presentation/base/BaseListDiffCallback.java

```diff
@@ -0,0 +1,16 @@
+package ru.yandex.practicum.contacts.presentation.base;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.DiffUtil;
+
+public class BaseListDiffCallback <T extends ListDiffInterface> extends DiffUtil.ItemCallback<T> {
```

Comment:
🍏 Лучше добавить в ограничение параметра BaseListDiffCallback дженерик-тип:
```suggestion
public class BaseListDiffCallback <T extends ListDiffInterface<T>> extends DiffUtil.ItemCallback<T> {
```